Overview.src.html

```diff
@@ -7147,12 +7146,13 @@ <h4 id="processing-a-keyframe-object">Processing a <code>Keyframe</code> object<
 1.  If the user agent supports animation of the 'float'
     CSS property, replace "float" in
     <var>supported properties</var> with "cssFloat".
-1.  If the user agent supports animation of the <a
-    href="http://www.w3.org/TR/SVG2/pservers.html#StopElementOffsetAttribute"
-    class="externalDFN">offset</a> attribute
-    of <a href="http://www.w3.org/TR/SVG2/pservers.html#StopElement"
-      class="externalDFN">SVG stop elements</a> [[!SVG2]], replace "offset" in
-    <var>supported properties</var> with "svgOffset".
+1.  For each DOM attribute that the user agent supports animation of on
+    SVG elements, replace the attribute in <var>supported properties</var>
+    with a name generated by:
+
+    1.  capitalizing the initial letter of the attribute name
```

Comment:
We probably need a pointer to a definition of ASCII capitalization.
Nit: we should end this point with ", and"